### PR TITLE
Cleaned up README.md

### PR DESCRIPTION
**Pull request by Daffehh**
Moved around the userscript mode to bring attention to it as the easier to use mode, made it's instructions mimic the headless mode, and a couple of captializations

### CODE REVIEW

**Comment by Daffehh** [comment]
@meepen fixed it

File: README.md [hunk 1]
```diff
@@ -1,20 +1,48 @@
...
+
+
+## Userscript Mode
+
+Userscript Mode is a version of this bot that is ran as a script on top of your browser, while having a tab with the game open. It is generally easier to use.
+
+Here are the steps to use: 
+
+- Download and install the TamperMonkey or GreaseMonkey addon for your browser of choice - https://tampermonkey.net/
+- Open this link: https://raw.githubusercontent.com/meepen/salien-bot/master/index.user.js with TamperMonkey installed; it should open up a tab that prompts you to install the script
```
Comment (meepen) [inline README.md]: Can you change this back to https://github.com/meepen/salien-bot/raw/master/index.user.js

Comment (Daffehh) [reply README.md]: changed back

File: README.md [hunk 2]
```diff
@@ -1,20 +1,48 @@
...
+
+Here are the steps to use: 
+
+- Download and install the TamperMonkey or GreaseMonkey addon for your browser of choice - https://tampermonkey.net/
+- Open this link: https://github.com/meepen/salien-bot/raw/master/index.user.js with TamperMonkey installed; it should open up a tab that prompts you to install the script
+- And finally, go to the Saliens game page!
+
+It should be running now!
+
+This version should automatically update if I decide to update version numbers properly too. :)
```
Comment (meepen) [inline README.md]: I've removed the update url since it can cause security issues

File: README.md [hunk 3]
```diff
@@ -1,20 +1,48 @@
...
+Also, a note: playing this game AT ALL will not net you better rewards! It's only the time you have been on the page. There's no need to waste computer resources. :)
+
+
+## Userscript Mode
+
+Userscript Mode is a version of this bot that is ran as a script on top of your browser, while having a tab with the game open. It is generally easier to use.
+
+Here are the steps to use: 
+
+- Download and install the TamperMonkey or GreaseMonkey addon for your browser of choice - https://tampermonkey.net/
```
Comment (meepen) [inline README.md]: we can't support greasemonkey, can you remove that?

**Comment by Daffehh** [comment]
@meepen did all your requested changes, sorry for making you have to wade through this to find my errors :)

**Pull request merged**